src/content/pages/volunteers.mdx

```diff
@@ -38,83 +44,86 @@ This is a very people-first position and will require spending a considerable ti
 
-* Helping speakers in preparing their computers for the talks
+* Helping speakers prepare their computers for the talks
 * Introducing the speakers to the audience
-* Signalling time left to the speakers
+* Signalling the speakers how much time is left
 * Making sure talks start and finish on time
-* Moderating Q&A sessions, assisting with passing the microphones and read out questions from remote attendees
+* Moderating Q&A sessions, assisting with passing the microphones and reading out questions from remote attendees
 
 ## Setup/Teardown
 
-* Putting up signage in hallways for various provisions (Bathroom, Quiet Room, Childcare etc.)
-* Also on the last day (Friday afternoon for the main conference, Sunday for the sprints) to clear up the venue afterward
-* This job involves walking and carrying items, some of which may be heavy.
+* Putting up signage in hallways for various provisions (Bathroom, Quiet Room, Childcare, etc.)
+* On the last day (Friday afternoon for the main conference, Sunday for the sprints), there will be a venue cleanup afterwards
+* This job involves walking and carrying items, some of which may be heavy
 
-## Greeter / Badge check
+## Greeter / Badge Check
 
-* Checking attendees on the venue for badges.
-* Send people to the registration desk if they don't have one and are not from the venue (e.g. AV team, catering, etc.).
-* Report people that don't have badges and don't want to go to the registration desk.
+* Checking attendees at the venue for badges
+* Sending people to the registration desk if they don't have a badge, and not from the venue (e.g. AV team, catering, etc.)
+* Reporting people who don't have badges and refuse to go to the registration desk
 
 ## Runners and Ad-hoc Activities
 
 There are always some tasks which we simply cannot plan beforehand. Based on what's needed in the moment, you can be expected to do one or more of the following (not an exhaustive list):
 
-* Checking rooms to ensure all the necessary resources are in place
-* Helping out with Registration Desk duties, T-Shirt Handout, or Room Manager duties
+* Checking the rooms to ensure all the necessary resources are in place
+* Helping out with Registration Desk, T-shirt Handout, or Room Manager duties
 * Moving chairs, tables, or poster walls
-* Arranging transfers between venue and airport, etc. (Rarely)
+* Arranging transfers between venue and airport, etc. (rarely)
+
 
 # How to sign up
 
-We are currently still working out the latest details for the volunteering program. (We are trying to get more perks enabled for your volunteering time).
-Until then, you can sign up for volunteering by sending an email to [volunteers@europython.eu](mailto:volunteers@europython.eu) .
+After you fill out the form below and acknowledge our [Data Privacy Policy](https://forms.gle/PCP7bXvdH1QhDBPn6), we will reach out to you with a link to join the EuroPython Discord server.
```

Comment:
This is not correct. The request for Data Privacy Policy is part of the form in the middle of the next paragraph. But filling it in will not lead to a discord invite.
And we will not invite people right away. There will be a selection based on the next form.

That said the form needs to be filled in.

Reply:
Sounds better.